openapi/crmaas/crmass.json

```diff
@@ -0,0 +1,243 @@
+{
+  "openapi": "3.1.0",
+  "x-stoplight": {
+    "id": "iyzxipdhj3szx"
+  },
+  "info": {
+    "title": "crmaas",
+    "version": "1.0"
+  },
+  "servers": [
+    {
+      "url": "https://prod.apigateway.co/crm",
+      "description": "Production"
+    },
+    {
+      "url": "https://demo.apigateway.co/crm",
+      "description": "Demo"
+    },
+    {
+      "url": "http://localhost:11001/crm",
+      "description": "Local"
+    },
+    {
+      "url": "{local}/crm",
+      "description": "Local"
+    }
+  ],
+  "paths": {
+    "/{namespace}/{resourceTypeCode}/meta/fields": {
+      "parameters": [
+        {
+          "schema": {
+            "type": "string"
+          },
+          "name": "namespace",
+          "in": "path",
+          "required": true,
+          "description": "The namespace, PID/AGID"
+        },
+        {
+          "schema": {
+            "type": "string"
+          },
+          "name": "resourceTypeCode",
+          "in": "path",
+          "required": true,
+          "description": "The resourceTypeCode, here Contact"
+        }
+      ],
+      "get": {
+        "summary": "Get field schema",
+        "tags": [],
+        "responses": {
+          "200": {
+            "description": "Schema found",
+            "content": {
+              "application/json": {
```

Comment:
I thought we were going to return an [OAS schema-object](https://spec.openapis.org/oas/v3.1.0#schema-object). Since it is a standard format we might be able to skip documenting the individual fields for now and instead link to the standard and provide an example.